src/lib/components/investment-card.svelte

```diff
@@ -34,15 +34,27 @@
 		portfolio: Portfolio
 		viewOnly?: boolean
 		index: number
+		hidden: boolean
+		focused: boolean
 		openTransaction?: (investment: InvestmentWithColorIndex, transaction?: Transaction) => void
+		toggleHide: (e: Event) => void
```

Comment:
I don't think you need the event here, right?
```suggestion
		toggleHide: () => void
```